query_test.go

```diff
@@ -3220,6 +3220,43 @@ FROM
 				{"false", "boolean"},
 			},
 		},
+
+		// subquery expr
+		{
+			name:         "SubqueryExpr(subquery_type=SCALAR) at SELECT",
```

Comment:
I want to express `name` parameter without using symbols such as `()=` .
e.g.) `subquery expression with scalar type at SELECT`

Reply:
ok, fixed!